chatmaild/src/chatmaild/doveauth.py

```diff
@@ -46,7 +46,7 @@ def is_allowed_to_create(config: Config, user, cleartext_password) -> bool:
         len(localpart) > config.username_max_length
         or len(localpart) < config.username_min_length
     ):
-        if localpart != "echo":
+        if localpart != "echo" and localpart != "hello":
```

Comment:
the pattern i prefer for this is: 
```suggestion
        if localpart not in ("echo", "hello"): 
```